defaults/main.yml

```
@@ -1 +1,13 @@
 clamav_cron_frequency: weekly
+clamav_scan_exclude_directories:
+  - ^/dev/
+  - ^/sys/
+  - ^/proc/
+  - ^/var/spool/clamav
+clamav_scan_move: false
+clamav_scan_copy: false
+clamav_scan_quarantine_dir: '/var/spool/clamav'
+clamav_scan_quarantine_owner: root
+clamav_scan_quarantine_group: root
+clamav_scan_quarantine_mode: '0750'
+clamav_scan_extra_flags: []
```

We prefer to alphabetize wherever possible:
```suggestion
clamav_scan_copy: false
clamav_scan_exclude_directories:
  - ^/dev/
  - ^/proc/
  - ^/sys/
  - ^/var/spool/clamav
clamav_scan_extra_flags: []
clamav_scan_move: false
clamav_scan_quarantine_dir: '/var/spool/clamav'
clamav_scan_quarantine_group: root
clamav_scan_quarantine_mode: '0750'
clamav_scan_quarantine_owner: root
```